setup.py

```diff
@@ -1,34 +1,54 @@
 """Setup script for temporal-sqlalchemy"""
 
 import sys
-from setuptools import setup, find_packages
+import setuptools
 
-if sys.version_info < (3, 5):
-    raise Exception('Python version < 3.5 are not supported.')
+if sys.version_info < (3, 3):
+    raise Exception('Python version < 3.3 are not supported.')
 
 # Get version information without importing the package
 __version__ = None
-exec(open('temporal-sqlalchemy/version.py', 'r').read())  # pylint: disable=exec-used
+exec(open('temporal_sqlalchemy/version.py', 'r').read())
 
+SHORT_DESCRIPTION = 'Temporal Extensions for SQLAlchemy ORM'
+LONG_DESCRIPTION = open('README.rst', 'r').read()
 
-setup(
+DEPENDENCIES = [l.strip() for l in open('requirements.txt', 'r')]
+if sys.version_info < (3, 5):
+    DEPENDENCIES.append('typing>=3.5.2,<4.0.0')
```

Comment:
Annoyingly, it provides `typing.Type` for Python < 3.5 but not for 3.5.0<=Python<3.5.2